File: pubnub-gson/src/main/java/com/pubnub/api/endpoints/channel_groups/AllChannelsChannelGroup.java
package com.pubnub.api.endpoints.channel_groups;

import com.pubnub.api.endpoints.Endpoint;
import com.pubnub.api.models.consumer.channel_group.PNChannelGroupsAllChannelsResult;

public interface AllChannelsChannelGroup extends Endpoint<PNChannelGroupsAllChannelsResult> {
    AllChannelsChannelGroup channelGroup(String channelGroup);
}

File: pubnub-kotlin/pubnub-kotlin-api/src/jvmMain/kotlin/com/pubnub/api/endpoints/channel_groups/AllChannelsChannelGroup.kt
package com.pubnub.api.endpoints.channel_groups

import com.pubnub.api.Endpoint
import com.pubnub.api.models.consumer.channel_group.PNChannelGroupsAllChannelsResult

/**
 * @see [PubNub.listChannelsForChannelGroup]
 */
interface AllChannelsChannelGroup : com.pubnub.kmp.endpoints.channel_groups.AllChannelsChannelGroup, Endpoint<PNChannelGroupsAllChannelsResult> {
    val channelGroup: String
}
